development/test-log-collector/README.md

```diff
@@ -0,0 +1,62 @@
+# Test Log Collector
+
+## Overview
+
+The purpose of the Test Log Collector is to gather logs from the testing pods and to send them to the appropriate slack channels.
+It is intended to be run after the Octopus ClusterTestSuite, as it uses pods labels from Octopus. Test Log Collector will send logs only from the newest, finalised ClusterTestSuite.
+
+## Requirements
```

Comment:
```suggestion
## Prerequisites
```

Reply:
As per the [template](https://github.com/kyma-project/community/blob/master/guidelines/repository-guidelines/repository-template/README.md).